README.md

```diff
@@ -125,6 +125,9 @@ When set to `true`, the Jekyll site will be built but not published
 ### pre_build_commands
 Commands to run prior to build and deploy. Useful for ensuring build dependencies are up to date or installing new dependencies. For example, use `apk --update add imagemagick` to install ImageMagick.
 
+### pre_publish_commands
+Commands to run prior to publish.
```

Comment:
Perhaps including more information here such as what's outlined in the `pre_build_commands` section